GitCommands/CommitHelper.cs

```diff
@@ -4,74 +4,83 @@ namespace GitCommands
 {
     public static class CommitHelper
     {
-        public static void SetCommitMessage(GitModule module, string commitMessageText, bool amendCommit)
+        public static string GetCommitMessagePath(GitModule module)
```

Comment:
Any API that pass an instance of `IGitModule` around is bad.
It looks like we only need `WorkingDirGitDir` and `CommitEncoding`, let's pass them instead